docs/source/hyper_log_log.rst

```diff
@@ -7,26 +7,49 @@ If the ONLY use case for sketching is counting uniques and merging, the HLL sket
 This implementation offers three different types of HLL sketch, each with different trade-offs with accuracy, space and performance. 
 These types are specified with the target_hll_type parameter.
 
-In terms of accuracy, all three types, for the same lg_config_k, have the same error distribution as a function of n, the number of unique values fed to the sketch.
-The configuration parameter `lg_config_k` is the log-base-2 of `K`, where `K` is the number of buckets or slots for the sketch.
+In terms of accuracy, all three types, for the same lg_config_k, have the same error distribution as a function of `n`, the number of unique values fed to the sketch.
+The configuration parameter `lg_config_k` is the log-base-2 of `k`, where `k`` is the number of buckets or slots for the sketch.
```

Comment:
For consistency with the files above, is it worth switching the backticks to double backticks? eg `k` to ``k``

Reply:
This is done